packages/cubejs-testing/src/birdbox.ts

```diff
@@ -449,9 +455,11 @@ export async function startBirdBoxFromCli(
     }
     await pausePromise(10 * 1000);
   } catch (e) {
+    process.stdout.write(`Error spawning cube: ${e}`);
```

Comment:
Maybe you need to add "\n" symbol in the end of the string?

Reply:
Good catch!